### ISSUE TITLE
Low GPU usage with CUDA

### ISSUE THREAD

**Issue by shipurjan** [kha-white/mokuro#62]
My GPU usage is low, but my CPU usage is at 100%. Just wanted to ask if that's normal, or should GPU usage be higher?

`torch.cuda.is_available()` returns `true` etc.

```
Each of the paths above will be treated as one volume. Continue? [yes/no]
yes
2023-06-24 00:59:45.389 | INFO     | mokuro.run:run:43 - Processing 1/7: C:\Users\user\Downloads\nausicaa\01
Processing pages...:   0%|[00:00<?, ?it/s]
2023-06-24 00:59:45.403 | INFO     | mokuro.manga_page_ocr:__init__:30 - Initializing text detector
2023-06-24 00:59:45.796 | INFO     | manga_ocr.ocr:__init__:13 - Loading OCR model from kha-white/manga-ocr-base
2023-06-24 00:59:48.340 | INFO     | manga_ocr.ocr:__init__:19 - Using CUDA
2023-06-24 00:59:50.981 | INFO     | manga_ocr.ocr:__init__:29 - OCR ready
Processing pages...:20% |████████  | 28/137 [02:09<10:56,  6.02s/it]
``` 

![image](https://github.com/kha-white/mokuro/assets/57770826/78f36e60-157c-47dc-a340-600275a14a42)

**Comment by rikafuruda** [comment]
I am also having this issue, did you figure out a solution?

**Comment by shipurjan** [comment]
> I am also having this issue, did you figure out a solution?

No, but judging from my screenshot it seems like I was CPU bottlenecked. See if your CPU usage is 100%

**Comment by kha-white** [comment]
Text detector now uses GPU if available in mokuro>=0.2.1, so it should be better now.
